docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia-personalization.md

```diff
@@ -0,0 +1,355 @@
+
+{% endinfo_block %}
+
+## Prerequisites
+
+1. [Integrate Algolia](/docs/pbc/all/search/{{page.version}}/base-shop/third-party-integrations/algolia/integrate-algolia.html)
+2. [Configure Algolia](/docs/pbc/all/search/{{page.version}}/base-shop/third-party-integrations/algolia/configure-algolia.html)
+3. Add products to your Algolia search indexes
+
+## Update Spryker Shop
+
+1. Install new Spryker packages:
+
+```bash
+composer require --with-dependencies spryker-shop/traceable-event-widget:^1.0.2
+```
+if the command does now work, try it with `--with-all-dependencies` flag instead.
+
+
+2. Update Spryker packages:
+
+```bash
+composer update --with-dependencies spryker-shop/cart-page:^3.45.0 spryker-shop/catalog-page:^1.28.0 spryker-shop/checkout-page:^3.32.1 \
+spryker-shop/home-page:^1.2.0 spryker-shop/payment-page:^1.5.0 spryker-shop/product-detail-page:^3.23.0 spryker-shop/product-group-widget:^1.10.1 \
+spryker-shop/product-review-widget:^1.16.1 spryker-shop/product-set-detail-page:^1.11.0 spryker-shop/quick-order-page:^4.10.1 \
+spryker-shop/shop-ui:^1.82.0
+```
+if the command does now work, try it with `--with-all-dependencies` flag instead.
+
+### Enable new features
+
+1. Update project config:
+```php
+// config_default.php
+
+$config[KernelAppConstants::TENANT_IDENTIFIER]
+    //..   
+    = getenv('SPRYKER_TENANT_IDENTIFIER') ?: '';
+```
+
+2. Enable a widget that triggers events for user actions and sends them to [Algolia Insights](https://www.algolia.com/doc/guides/sending-events/getting-started/):
+```php
+// src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php
+
+class ShopApplicationDependencyProvider extends SprykerShopApplicationDependencyProvider
+{
+
+    protected function getGlobalWidgets(): array
+    {
+        return [
+            //...
+            SprykerShop\Yves\TraceableEventWidget\Widget\TraceableEventWidget::class
+        ];
+}
+```
+
+3. Enabled the plugin that generates an anonymous token for guest users in the session:
+```php
+// src/Pyz/Yves/EventDispatcher/EventDispatcherDependencyProvider.php
+
+class EventDispatcherDependencyProvider extends SprykerEventDispatcherDependencyProvider
+{
+
+    protected function getEventDispatcherPlugins(): array
+    {
+        return [
+            //...
+            new Spryker\Yves\Customer\Plugin\EventDispatcher\AnonymousIdSessionAssignEventDispatcherPlugin(),
+        ];
+}
+```
+
+4. Build JavaScript assets for Yves using one of the following commands:
+```bash
+npm ci && npm run yves
+```
+
+```bash
+console frontend:project:install-dependencies && console frontend:yves:build
+```
+
+5. Install required `search-insights` dependency:
+```bash
+npm i search-insights`.
+
+
+### Check your Yves's compatibility with new functionality
+
+In case of customizations, your codebase could have updated Yves templates on the project level (`src/Pyz/Yves/`).
+It could be a reason that some events won't be triggered or triggered with incorrect data.
```

Comment:
What do I need to do in this case?

Reply:
This paragraph is part of "Check your Yves's compatibility with new functionality", you need to follow the steps from the next sub-sections
- Run the project
- Check triggered events and their payload